webpack.config.js

```diff
@@ -28,6 +28,11 @@ module.exports = function (env) {
                 },
 
                 {
+                    test: /\.json$/,
```

Comment:
I've just read "Note: Since webpack v2, JSON files will work by default. You might still want to use this if you use a custom file extension." I'll fix it, sorry, should read docs next time